docs/configuration-parameters.md

```diff
@@ -142,7 +142,7 @@ This section provides the details of all the configurations required for CarbonD
  
 | Parameter | Default Value | Description |
 |----------------------------------------|--------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| spark.driver.memory | 1g | Amount of memory to use for the driver process, i.e. where SparkContext is initialized. NOTE: In client mode, this config must not be set through the SparkConf directly in your application, because the driver JVM has already started at that point. Instead, please set this through the --driver-memory command line option or in your default properties file. |
+| spark.driver.memory | 1g | Amount of memory to use for the driver process. |
 | spark.executor.memory | 1g | Amount of memory to use per executor process. |
```

Comment:
We can replace the description by : Amount of memory to be used per executor process.